zipper/protocols/irondb/irondb_group.go

```diff
@@ -520,10 +506,10 @@ func (c *IronDBGroup) Find(ctx context.Context, request *protov3.MultiGlobReques
 		logger.Debug("will do find query",
 			zap.Int64("accountID", c.accountID),
 			zap.String("query", query),
-			zap.String("prefix", c.graphite_prefix),
+			zap.String("prefix", c.graphitePrefix),
 		)
 		stats.FindRequests++
-		find_result, err := c.client.GraphiteFindMetrics(c.accountID, c.graphite_prefix, query, nil)
+		find_result, err := c.client.GraphiteFindMetrics(c.accountID, c.graphitePrefix, query, nil)
```

Comment:
still one instance of a snake_case :)  (that's the only place where I've noticed it)

Reply:
Thanks! Fixed now. I checked all "_" - should be no more. :)